### PR TITLE
Fixes dead link - https://wger.de/

### PR DESCRIPTION
**Pull request by skadogg**

### CODE REVIEW

**Review by kokomo123 — APPROVED** [review]
TYVM, thanks for fixing 💯

File: software/wger.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ platforms:
 tags:
   - Miscellaneous
 source_code_url: https://github.com/wger-project/wger
-demo_url: https://wger.de/en/dashboard
+demo_url: https://github.com/wger-project/wger/tree/master/extras/docker/demo
```
Comment (nodiscc) [inline software/wger.yml:12]: https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/PULL_REQUEST_TEMPLATE.md

> Demo links should only be used for interactive demos

The previous link was fine

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/wger.yml (hunk 1 above)
Comment (skadogg) [reply software/wger.yml:12]: Weird. The link didn't seem to work when I checked it the other day.

**Comment by Ki-er** [comment]
Both links seem to be operational.

**Pull request closed without merging**